package.json

```
@@ -25,7 +25,7 @@
   "bugs": {
     "url": "https://github.com/LorenzoDrudi/ephemeral-github-runner-action/issues"
   },
-  "homepage": "https://github.com/LorenzoDrudi/ephemeral-github-runner-action/blob/main/README.md",
+  "homepage": "https://github.com/LorenzoDrudi/ephemeral-github-runner-action#readme",
```

what's the difference between these two?

idk but this could also work?
```suggestion
  "homepage": "https://github.com/LorenzoDrudi/ephemeral-github-runner-action/blob/main/README.md#javascript-github-action-to-deploy-and-destroy-self-hosted-runners",
```

---

The one I use doesn't open the readme file but goes in the main page of the repo on the readme in the bottom of the page.
Maybe it's better to open the file